app/javascript/css/components/track.css

```diff
@@ -0,0 +1,17 @@
+@import "../styles";
```

Comment:
What does `&` mean here?

Reply:
`&` is the CSS3 nesting operator. Whereas in sass it's optional (and normally not added), in CSS3 it's compulsory.